package/MDAnalysis/coordinates/ParmEd.py

```diff
@@ -163,8 +163,9 @@ def convert(self, obj):
         try:
             import parmed as pmd
         except ModuleNotFoundError:
-            raise ValueError('Parmed is required for ParmEdConverter but '
-                             'is not installed.')
+            raise ImportError('ParmEd is required for ParmEdConverter but '
```

Comment:
Do we `import coordinates.ParmEd` anywhere, i.e., is it guaranteed that this does not break the installation for anyone _not_ having ParmEd?